### DISCUSSION TITLE
Getting started

### DISCUSSION THREAD

**Discussion by NickElkington** [Short-bus/pilomar#69] [General]
Hi - saw this a few weeks ago and love the idea so I am getting into it now. I have some relevant experience using arduinos to drive motors and control boards for optical systems but this is my first time with a raspberry pi. I have my printer producing the foundation for me and most of the electronics ordered but my only point of confusion was the 12v 8a power supply. I'm not sure exactly what interfacing I'd need to provide power to it. Would you recommend something like this: https://www.amazon.co.uk/Velain-Universal-Regulated-Transformers-Surveillance/dp/B01N6O0LOH/ref=asc_df_B01N6O0LOH/?tag=googshopuk-21&linkCode=df0&hvadid=521605924482&hvpos=&hvnetw=g&hvrand=527070200641254262&hvpone=&hvptwo=&hvqmt=&hvdev=c&hvdvcmdl=&hvlocint=&hvlocphy=1006976&hvtargid=pla-792319236135&mcid=49ebf3178c713d788c4d3b24a7e04d93&th=1

I'm really looking forward to getting into this! Is there any other platform for people who have built these to discuss ie, Facebook group, discord or so?

Many thanks and I look forward to building and tinkering with this and hopefully once its up and running can contribute more!

Nick

**Comment by Short-bus** [comment]
Hi Nick,
The power supply can be pretty straight forward. 
I always use an official Raspberry Pi power supply for the RPi itself, they seem to provide reliable power to the computer.

The larger 12V power supply is only to drive the stepper motors. So you need some way to provide 12V at 8A into the motorcontroller PCB, that feeds it directly into the two DRV8825 chips and from there it's sent to the coils of the motors. 
Regarding the choice of power supply. The chassis type of power supply you found on Amazon will be fine I'm sure.

I went even simpler and found some on ebay which were just like the traditional power bricks for laptops. Very generic, but I chose a domestic supplier just to reduce the chances of buying anything sub-standard.
Something similar to this...
https://www.ebay.co.uk/itm/375190585740

They have the barrel connectors on them, I just found a matching female connector lead on ebay and screwed that into the PCB. 
https://www.ebay.co.uk/itm/266249476457

Obvs the usual warnings always apply... trust the maker of the power supply and make sure it's protected from the elements and fingers when in use. 

I think a couple of builders now have made more elaborate circuits where they ONLY have the 12V supply and then use a buck-converter to pull out 5V for the RPi. That's an option too if you're happy with making that sort of circuit up.

If you're building your own PCB from scratch I can suggest some updates to the schematic for you. I'll be incorporating feedback from everyone into a summer revision to the PCB design in the GitHub repository, there's nothing official yet, but I can make a couple of improvement suggestions already. 

Good luck with the build, and keep in touch with questions/progress/results!
Matt.

**Comment by Short-bus** [comment]
Regarding discussion groups for the project. Send me a PM via the Instructables website (we can't PM on GitHub can we?). We're just testing a Discord group at the moment.
